code/__DEFINES/mobs.dm

```diff
@@ -418,6 +418,8 @@ GLOBAL_LIST_INIT(xenoupgradetiers, list(XENO_UPGRADE_BASETYPE, XENO_UPGRADE_INVA
 
 //Xeno Defines
 
+#define MIN_HIVE_SIZE_FOR_QUEEN 6
```

Comment:
this should be hive datum per hive var

Reply:
How do we work it at roundstart, before any xenos have spawned?